aal_purchase_form/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': 'aal_Purchase_Form',
+    'version': '12.0.1.0.1',
```

Comment:
```suggestion
    'version': '12.0.1.0.0',
```
First version start at 12.0.1.y.z   when Fix bug after merged we will run number at 'z'
Add function we will run number 'y'